relentless/optimize/method.py

```diff
@@ -145,15 +153,33 @@ class LineSearch:
 
         \mathbf{d} = \mathbf{x}_{end} - \mathbf{x}_{start}
 
-        \hat{\mathbf{d}} = \frac{\mathbf{d}}{\lVert\mathbf{d}\rVert}
+        \mathbf{\hat{d}} = \frac{\mathbf{d}}{\lVert\mathbf{d}\rVert}
+
+    :math:`\mathbf{X}=\left[X_1,\ldots,X_n\right]` is defined as the scaling
+    parameters for the gradient such that :math:`y_i=\frac{x_i}{X_i}`. By
+    default, the variables are left unscaled (:math:`X_i=1`).
+
+    With scaling, the gradient of the function becomes:
+
+    .. math::
+
+        \nabla f\left(\mathbf{y}\right) = \left[{X_1}^2 \frac{\partial f}{\partial x_1},
+                                                \cdots,
+                                                {X_n}^2 \frac{\partial f}{\partial x_n}\right]
```

Comment:
Shouldn't there be no `^2` here?
```
df/dy = df/dx dx/dy = X df/dx
```
if `x = Xy`

Reply:
Mike is correct, only one factor of X here. Since we don't actually need the scale factors for the LineSearch, let's remove anything involving the scaled variables and scaled gradient.